config.ts

```diff
@@ -0,0 +1,3 @@
+module.exports = {
```

Comment:
Det hade varit nice att använda https://www.npmjs.com/package/dotenv istället, där kan man även stoppa in MongoDB strängen och vilken PORT som ska användas osv, men det är mer relevant när vi ska hosta allt